Model/Export/AbstractExportManager.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace StoreKeeper\StoreKeeper\Model\Export;
+
+use Magento\Framework\Locale\Resolver;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Store\Api\StoreConfigManagerInterface;
+
+abstract class AbstractExportManager
+{
+    private Resolver $localeResolver;
+    private StoreManagerInterface $storeManager;
+    private StoreConfigManagerInterface $storeConfigManager;
+
+    public function __construct(
+        Resolver $localeResolver,
+        StoreManagerInterface $storeManager,
+        StoreConfigManagerInterface $storeConfigManager
+    ) {
+        $this->localeResolver = $localeResolver;
+        $this->storeManager = $storeManager;
+        $this->storeConfigManager = $storeConfigManager;
+    }
+
+    /**
+     * @param array $headersPaths
+     * @param array $headersLabels
+     * @return array
+     */
+    public function getMappedHeadersLabels(array $headersPaths, array $headersLabels): array
+    {
+        return array_combine($headersPaths, $headersLabels);
+    }
+
+    /**
+     * @return string
+     */
+    public function getCurrentLocale(): string
```

Comment:
this should be read from the storekeeper settings

Reply:
@ikulis Already corrected in commit for 23rd of June - https://github.com/storekeeper-company/magento2-plugin/commit/9bd36578d720e8febde5dc5f38e687958d30fbca
I dunno why it is not registered in github pr